tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Connecting a Haskell Backend to a Purescript Frontend
+published: not-yet
+ghc: 7.10.3
+purs: 0.10.5
+lts: 7.17
+libraries: purescript-bridge
+language: haskell purescript
+author-name: Javier Casas Velasco
+description: In this tutorial we will implement a way to extend the types in the Haskell backend to the Purescript frontend while maintaining consistency and simplifying communication.
+---
+# Connecting a Haskell Backend to a Purescript Frontend
+## Introduction
+At Stackbuilders we are working on a full-stack app with CollegeVine https://www.collegevine.com/ using functional languages.
+We have a Haskell backend, based on Servant, that manipulates the database and offers some endpoints to a Purescript frontend,
```

Comment:
@javcasas Could we remove some commas here in order to make reading more fluid? What about:

We have a backend written in [Servant](https://github.com/haskell-servant/servant) that manipulates the database and offers some endpoints to a PureScript frontend ...

Also, I think would be nice to include some links to some of the packages you are referencing.